sprints/users/adapters.py

```diff
@@ -13,4 +13,4 @@ def is_open_for_signup(self, request: HttpRequest):
 
 class SocialAccountAdapter(DefaultSocialAccountAdapter):
```

Comment:
@Agrendalath, do we want any Google account to be able to login and not just the ones under the `opencraft.com` domain? If so, some code changes (and corresponding tests) will be required.

Reply:
@Agrendalath, I tested this and found that it works correctly and throws an error. :+1:

Reply:
@lgp171188, this has been set up on the Google application side.